src/domain/LottoIncome.js

```diff
@@ -0,0 +1,59 @@
+const LottoCalculator = require('./LottoCalculator');
+
+const { LOTTO_AMOUNT } = require('../../utils/constants');
+
+class LottoIncome extends LottoCalculator {
+  #income;
+
+  constructor(inputs) {
+    super(inputs);
+
+    this.#income = '';
+    this.#calculateIncome();
+  }
+
+  getResult() {
+    return this.#income;
+  }
+
+  #roundUpFor() {
+    this.#income = Math.round(
+      (LOTTO_AMOUNT.reduce((acc, moneyUnit, index) => {
+        return acc + moneyUnit * this.getLottoCountScore()[index];
+      }, 0) /
+        this.payment.getMoney()) *
+        1000,
+    );
```

Comment:
위처럼 구현하게 된다면 계산식이 많아서 ```this.#income``` 변수에 어떤 값이 들어가는지 명확하게 보이지 않는 것 같아요. 변수로 계산을 쪼개서 값을 할당하면 더 가독성이 좋아지지 않을까요?

```suggestion
    const prize =  LOTTO_AMOUNT.reduce((acc, moneyUnit, index) => {
        return acc + moneyUnit * this.getLottoCountScore()[index];
      }, 0);
    const value = this.payment.getMoney()) * 1000;
    this.#income = Math.round(value)
    );
```

Reply:
코드 리뷰 정말 감사합니다 👍👍

> 위처럼 구현하게 된다면 계산식이 많아서 this.#income 변수에 어떤 값이 들어가는지 명확하게 보이지 않는 것 같아요. 변수로 계산을 쪼개서 값을 할당하면 더 가독성이 좋아지지 않을까요?

넵, 동의합니다 ㅜㅜ

추가적으로 prize와 value부분을 함수로 더 쪼겠으면 더 좋았을 거 같아요 ㅎㅎ